odk1-src/collect-offline-maps.rst

```diff
@@ -1,44 +1,68 @@
+Collect's :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, and :ref:`geoshape <geoshape-widget>` questions can be configured to display different map data. To use online basemaps, :ref:`set the basemap source and style <basemap-settings>` in :guilabel:`mapping-settings`. Users will need to be online to load questions with maps.
 
-Offline maps are useful for low-connectivity environments or to present custom geospatial data. Use them to display high-resolution imagery, annotated maps, heatmaps, and more. ODK Collect can display any map layer saved as a set of tiles in the `MBTiles format <https://github.com/mapbox/mbtiles-spec>`_. Tiles are images that represent a subset of a map. The only limitation is that tile data in Mapbox's pbf format are not supported.
+Offline layers are useful to present custom geospatial data layered over standard basemaps or as basemaps for low-connectivity environments. Use them to display high-resolution imagery, annotated maps, heatmaps, and more. ODK Collect can display any map layer saved as a set of tiles in the `MBTiles format <https://github.com/mapbox/mbtiles-spec>`_.
+
+.. warning::
+
+  - Vector MBTiles (with data in `Mapbox Vector Tile format (pbf) <https://github.com/mapbox/vector-tile-spec>`_) are only supported if :guilabel:`Mapbox` is selected as the :ref:`basemap source <basemap-settings>`.
+
+  - Vector MBTiles are currently displayed without styling. Each layer's lines are displayed in a different color picked by ODK Collect. This color will be the same across all devices using the same MBTiles file. Fills are not displayed.
 
 .. _offline-maps-quick-start:
 
 Offline maps quick start
 -------------------------
-#. From :ref:`Collect's settings <mapping-settings>`, change the Mapping SDK to Google Maps SDK
 #. :ref:`Get or create your MBTiles file <getting-map-tiles>` with `TileMill <https://tilemill-project.github.io/tilemill/>`_ or other software.
-#. :ref:`Transfer tiles to devices <transferring-offline-tiles>`. The MBTiles file must be placed in a sub-folder on your device in the :file:`/sdcard/odk/layers` folder.
-#. Open a :ref:`question that displays a map <location-widgets>`.
-#. Tap the layers icon and select your map
+#. :ref:`Transfer tiles to devices <transferring-offline-tiles>`. The MBTiles file must be placed on your device in the :file:`/sdcard/odk/layers` folder.
+#. Select your offline layer in the :ref:`reference layer settings <reference-layer-settings>`.
+#. Open a :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, or :ref:`geoshape <geoshape-widget>` questions.
+
+Your offline layer should be displayed. If it has transparency (PNG or PBF tiles only), the selected basemap will show through. If it does not have transparency or you are offline, only your offline layer will be displayed.
```

Comment:
MBTiles files typically contain
    `metadata <https://github.com/mapbox/mbtiles-spec/blob/master/1.3/spec.md#metadata>`
    that specifies the range of zoom levels in
    which they are visible.  If you are viewing at an appropriate
    zoom level, your offline layer should be displayed.  If you
    don't see it, you might need to zoom in or out until the
    zoom level is in the range specified by the MBTiles file.

    If the tileset has transparency...